dev-client/src/screens/LocationDashboardScreen.tsx

```diff
@@ -51,18 +51,9 @@ export const LocationDashboardScreen = ({siteId, coords}: Props) => {
   );
 
   const appBarRightButton = useMemo(() => {
-    // display add button if no site associated with location
-    if (!siteId) {
-      return (
-        <AppBarIconButton
-          name="add"
-          onPress={() => navigation.navigate('CREATE_SITE', {coords})}
-        />
-      );
-    }
-
-    // display nothing if user does not own the site / is not manager
-    if (userRole === null || !isSiteManager(userRole)) {
+    // display nothing if no site associated with location or
+    // user does not own the site / is not manager
+    if (!siteId || userRole === null || !isSiteManager(userRole)) {
```

Comment:
(Aside) Question: I've mostly seen "site owner" in our code so far. 
@tm-ruxandra Do you know if we have standardized terminology of "site owner" or "site manager"?

Reply:
Unfortunately there's a terminology collision happening here.

For unaffiliated sites, the user responsible for the site is referred to as "site owner" in (most) code and "site manager" in product documents. However, neither of those is consistent with the usage in the helper method here, which defines that either "site owner" or "project manager" is a "site manager". This is something I hope we can standardize on the frontend in the future to avoid precisely this sort of confusion :)

Reply:
Gotcha -- so sounds like today we mostly use 
- "site owner" for unaffiliated sites
- "project manager" for affiliated sites, because they adopt the user roles of their project
- "site manager" for any site, regardless of if it's affiliated or not

I think my ideal would be something like 
- "unaffiliated site owner"
- "affiliated site owner" / "project manager" (?) 
- "site owner"

Reply:
Maybe we can have a dedicated chat to go over this - I think there's still some misunderstanding going on about terminology. "Project manager" is a project-level role, not a site-level one; affiliated sites definitionally cannot have owners.